src/main/kotlin/com/github/suusan2go/kotlinfillclass/intentions/FillClassIntention.kt

```diff
@@ -68,7 +62,7 @@ class FillClassIntention : SelfTargetingIntention<KtValueArgumentList>(KtValueAr
             KotlinBuiltIns.isListOrNullableList(type) -> "emptyList()"
             KotlinBuiltIns.isSetOrNullableSet(type) -> "emptySet()"
             type.isMarkedNullable -> "null"
-            else -> null
+            else -> "TODO()"
```

Comment:
I think `TODO()` is not preferable.  Because this may cause unintentional runtime error if programmer doesn't notice it.

Reply:
Fixed